src/server/routes/api-routes.js

```diff
@@ -5,4 +5,24 @@ const router = express.Router();
 
 router.use('/items', items);
 
+router.use('/*', (req, res) => {
+  if (res.statusCode === 200 || res.statusCode === 304) {
+    return;
+  }
+  res.status(404).send('API Request > 404 - Page Not Found');
+});
+
```

Comment:
![image](https://cloud.githubusercontent.com/assets/20923534/18667261/cf7b100a-7f6a-11e6-8a9e-359c4b7c8e0d.png)
If I generate 400 error(Item not found error of GetById func), this func will be call but call other callback so occur some errors.
How can we solve itT_T